src/graphics.h

```diff
@@ -18,6 +18,15 @@
 #ifndef EP_GRAPHICS_H
 #define EP_GRAPHICS_H
 
+// Constant Definitions
+#ifndef _SIZE_RANDOM_BLOCKS_
+#define _SIZE_RANDOM_BLOCKS_ 4
+#endif
+
+#ifndef _SIZE_ARRAY_RANDOM_BLOCKS_
+#define _SIZE_ARRAY_RANDOM_BLOCKS_ 4800
```

Comment:
Please don't use #defines anymore, they are old-school. Also I wouldn't put them in the header.

The modern compile time constants are constexpr.
Just put them in a anonymous namespace (namespace { }) in graphics.cpp.
And follow the variable naming convention (random_blocks_size, random_blocks_array_size).